### PR TITLE
DS-8535: Framework and libs to apply RBAC at test suite

### PR DESCRIPTION
**Pull request by jkagliwal-px**
This PR solves - 
1. https://portworx.atlassian.net/browse/DS-8535
2. https://portworx.atlassian.net/browse/DS-8489

Build Torpedo Job -  https://jenkins.pwx.dev.purestorage.com/job/Users/job/Madan/job/platfrom-build-torpedo/155/

### CODE REVIEW

File: drivers/unifiedPlatform/stworkflows/workflowPlatformRBAC.go [hunk 1]
```diff
@@ -1,67 +0,0 @@
-package stworkflows
```
Comment (atrivedi-px) [inline drivers/unifiedPlatform/stworkflows/workflowPlatformRBAC.go:1]: Please don't delete this file, this is required for platform RBAC

File: tests/unifiedPlatform/platform/platform_test.go [hunk 2]
```diff
@@ -55,3 +58,37 @@ var _ = Describe("{PlatformOnboardingTest}", func() {
 		defer EndTorpedoTest()
 	})
 })
+
+var _ = Describe("{TestRbacForPds}", func() {
```
Comment (atrivedi-px) [inline tests/unifiedPlatform/platform/platform_test.go:62]: If this is a dummy test please move this to tests/unifiedPlatform/platform/platform_test_dummy.go

File: drivers/unifiedPlatform/stworkflows/workflowPlatformRBAC.go (hunk 1 above)
Comment (jkagliwal-px) [reply drivers/unifiedPlatform/stworkflows/workflowPlatformRBAC.go]: Added back

**Review by atrivedi-px — APPROVED** [review]
LGTM

**Review by Madanagopal19 — APPROVED** [review]
LGTM

**Pull request merged**